robyn/__init__.py

```diff
@@ -200,6 +201,13 @@ def set_request_header(self, key: str, value: str) -> None:
     def set_response_header(self, key: str, value: str) -> None:
         self.response_headers.set(key, value)
 
+    def exclude_response_headers(self, exclude_paths: Optional[List[str]]):
```

Comment:
this function name implies we are passing headers as a parameter. We should come up with a better name.